client/src/app/tabs/EngineProfile.js

```diff
@@ -0,0 +1,135 @@
+/**
```

Comment:
Let's add some basic test cases for the engine profile component at least.
* ensures it renders
* ensures it opens
* ensures it closes
* ensures it shows correct context (camunda, zeebe, dmn, forms)
* ....

What do you think?

Reply:
Sure, we can do that.

Reply:
OK I've added the tests.